src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+public class FigureSupplier extends ColorSupplier {
```

Comment:
```suggestion
public class FigureSupplier {
```
тут наслідування не зовсім підходить так як в цих класах різні цілі